### PR TITLE
feat: add useClearTextShortcut to setting api

### PR DESCRIPTION
**Pull request by ppken**
Added useClearTextShortcut to Settings api document.
Related PR: https://github.com/appium/WebDriverAgent/pull/952

### CODE REVIEW

File: docs/reference/settings.md [hunk 1]
```diff
@@ -41,3 +41,4 @@ Along with the common settings, the following driver-specific settings are avail
...
 | `webScreenshotMode` | `native` or `page` or `viewport` | Defines the screenshoting logic if the current context is set to a web one. The default value is `native`, which makes the driver to take screenshots from WDA, e.g. the whole device screen including status bars. The `page` mode tries to retrieve the screenshot of the whole active web page, while the `viewport` one only retrieves a shot of the visible viewport. |
+| `useClearTextShortcut` | `boolean` | Whether to use the fastest operation (using IOHIDEvent) to clear texts. In headless mode, simulator's keyboard won't show up anymore after clearing texts with this once in some cases. Defaults to true |
```
Comment (mykola-mokhnach) [inline docs/reference/settings.md:44]: you may also link the original issue for the better perspective.

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/reference/settings.md (hunk 1 above)
Comment (mykola-mokhnach) [inline docs/reference/settings.md:44]: `with this once` -> `using this method`

Comment (ppken) [reply docs/reference/settings.md]: Thank you! fixed in https://github.com/appium/appium-xcuitest-driver/pull/2490/commits/93ba79be7bc545c7b22bb194670305cef0c35f19

Comment (ppken) [reply docs/reference/settings.md]: Thank you! added in https://github.com/appium/appium-xcuitest-driver/pull/2490/commits/93ba79be7bc545c7b22bb194670305cef0c35f19

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**